tests/test_request.py

```diff
@@ -218,7 +217,7 @@ def test_request_request__server_down(self):
 
         with mock.patch('requests.sessions.Session.request') as mock_request:
             def request(*args, **kwargs):
-                raise ConnectionError("Server down!")
+                raise RequestException("Server down!")
```

Comment:
Why this change? `ConnectionError` still inherits from `RequestException`, so should still be caught. Maybe I don't understand something?

Reply:
Fair point, I will add one for Timeout as well to make sure that is caught instead of changing this to be the parent.